eng/common/scripts/stress-testing/java11-mvn-mariner-dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/openjdk/jdk:11-mariner as builder
```

Comment:
Will there be a template for each LTS version? Or could this be parameterized to handle all Java LTS versions (8, 11, 17, 21)